diff --git a/apps/ca.c b/apps/ca.c
index a7a5ab1ece..2e5cb38052 100644
--- a/apps/ca.c
+++ b/apps/ca.c
@@ -210,7 +210,7 @@ const OPTIONS ca_options[] = {
     {"noemailDN", OPT_NOEMAILDN, '-', "Don't add the EMAIL field to the DN"},
 
     OPT_SECTION("Signing"),
-    {"md", OPT_MD, 's', "Digest to use, such as sha256"},
+    {"md", OPT_MD, 's', "Digest to use; see openssl help for list"},
     {"keyfile", OPT_KEYFILE, 's', "The CA private key"},
     {"keyform", OPT_KEYFORM, 'f',
      "Private key file format (ENGINE, other values ignored)"},
